hook.go

```diff
@@ -39,11 +39,13 @@ type ElasticHook struct {
 }
 
 type message struct {
-	Host      string        `json:"host"`
-	Timestamp string        `json:"@timestamp"`
-	Message   string        `json:"message"`
-	Data      logrus.Fields `json:"data"`
-	Level     string        `json:"level"`
+	Host      string `json:"host,omitempty"`
+	Timestamp string `json:"@timestamp"`
+	File      string `json:"file,omitempty"`
+	Func      string `json:"func,omitempty"`
+	Message   string `json:"message,omitempty"`
+	Data      logrus.Fields `json:"data,omitempty"`
+	Level     string `json:"level,omitempty"`
```

Comment:
Can you please also run go fmt on this file? The alignment of the fields is not done according to `go fmt`.

Reply:
Done